Compilation errors on rustc 1.63

I'm trying to compile virtualization-rs using rustc 1.63, but I'm getting those errors:

```
$ cargo build
Compiling libc v0.2.132
Compiling block v0.1.6
Compiling malloc_buf v0.0.6
Compiling objc v0.2.7
Compiling virtualization-rs v0.1.2 (/Users/antek/dev/thirdparty/virtualization-rs)
error[E0054]: cannot cast as `bool`
--> src/base.rs:117:13
|
117 |             obj as BOOL
|             ^^^^^^^^^^^ unsupported cast

error[E0054]: cannot cast as `bool`
--> src/virtualization/virtual_machine.rs:196:20
|
196 |                 Ok(obj as BOOL)
|                    ^^^^^^^^^^^ unsupported cast

error[E0308]: mismatched types
--> src/virtualization/virtual_machine.rs:255:23
|
255 |             Ok(ret == 1i8)
|                       ^^^ expected `bool`, found `i8`

Some errors have detailed explanations: E0054, E0308.
For more information about an error, try `rustc --explain E0054`.
error: could not compile `virtualization-rs` due to 3 previous errors
FAIL: 101
```

Did anyone have a similar issue? Is there any quick fix for this?

--- a/src/base.rs
+++ b/src/base.rs
@@ -112,10 +112,7 @@ impl NSURL {
     }
 
     pub fn check_resource_is_reachable_and_return_error(&self) -> bool {
-        let b = unsafe {
-            let obj: Id = msg_send![*self.0, checkResourceIsReachableAndReturnError: NIL];
-            obj as BOOL
-        };
+        let b: BOOL = unsafe { msg_send![*self.0, checkResourceIsReachableAndReturnError: NIL] };
         b == YES
     }
 

--- a/src/virtualization/virtual_machine.rs
+++ b/src/virtualization/virtual_machine.rs
@@ -189,11 +189,11 @@ impl VZVirtualMachineConfiguration {
     pub fn validate_with_error(&self) -> Result<BOOL, NSError> {
         unsafe {
             let error = NSError(StrongPtr::new(0 as Id));
-            let obj: Id = msg_send![*self.0, validateWithError: &(*error.0)];
+            let obj: BOOL = msg_send![*self.0, validateWithError: &(*error.0)];
             if error.code() != 0 {
                 Err(error)
             } else {
-                Ok(obj as BOOL)
+                Ok(obj)
             }
         }
     }
@@ -252,7 +252,7 @@ impl VZVirtualMachine {
         if error.code() != 0 {
             Err(error)
         } else {
-            Ok(ret == 1i8)
+            Ok(ret)
         }
     }
 

